compose/cli/main.py

```diff
@@ -234,6 +234,7 @@ def ps(self, project, options):
 
         Options:
             -q    Only display IDs
+            --json    List containers in JSON format
```

Comment:
I'm wondering if we should actually support this as a top level option on `docker-compose` instead of an option on each subcommand.